wondrous-app/package.json

```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "lint:quiet": "next lint --quiet",
     "lint:fix": "prettier --write '**/*.{ts,tsx}' && eslint --fix .",
```

Comment:
haha nice

Reply:
it's really difficult to find errors among thousands of warnings :-)